shell/plugins/plugin.js

```diff
@@ -18,5 +20,57 @@ export default async(context) => {
     }
   });
 
-  await allHash(hash);
+  // Discover all of the UI Plugins CRs
+
+  let loadPlugins = true;
+
+  // Provide a mechanism to load the UI without the plugins loaded - in case there is a problem
+  if (context.route?.path === '/safeMode') {
```

Comment:
I tried this out (with an old backend... `v1/catalog.cattle.io.uiplugin` returning 598)
1) There's a long pause
2) `Could not load UI Plugins` error in logs (from ln 59 `shell/plugins.plugin.js`). Which i guess comes from the 598 above
3) generic 404 page is shown

The 404 seems odd, given we should be skipping things. There's a TODO in there, so don't know if this is still WIP

Reply:
This is still being wired into the backend